src/frontend/admin/src/services/api/models/CertificationDefinition.ts

```diff
@@ -0,0 +1,16 @@
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+
+export type CertificationDefinition = {
```

Comment:
Globally, you should rename `CertificationDefinition` into `CertificateDefinition`